spec/avram/migrator/alter_table_statement_spec.cr

```diff
@@ -89,6 +89,27 @@ describe Avram::Migrator::AlterTableStatement do
     built.statements[2].should eq "ALTER TABLE ONLY test_defaults ALTER COLUMN money SET DEFAULT '29.99';"
   end
 
+  it "changes defaults after changing the type" do
+    built = Avram::Migrator::AlterTableStatement.new(:users).build do
+      change_type id : Int64
+      change_default id : Int64, default: 54
+    end
+    built.statements.size.should eq 2
+    built.statements[0].should eq "ALTER TABLE users ALTER COLUMN id SET DATA TYPE bigint;"
+    built.statements[1].should eq "ALTER TABLE ONLY users ALTER COLUMN id SET DEFAULT '54';"
+  end
+
+  it "can change column nullability" do
+    built = Avram::Migrator::AlterTableStatement.new(:users).build do
+      forbid_nulls_for :id
+      allow_nulls_for :age
+    end
+
+    built.statements.size.should eq 2
+    built.statements[0].should eq "ALTER TABLE users ALTER COLUMN id SET NOT NULL;"
+    built.statements[1].should eq "ALTER TABLE users ALTER COLUMN age DROP NOT NULL;"
+  end
+
```

Comment:
oops! Looks like this snuck in here :joy:

Reply:
🤕 did the checkout from the wrong branch 😢